ui/apps/platform/src/Containers/NetworkGraph/external/ExternalEntitiesSideBar.tsx

```diff
@@ -0,0 +1,92 @@
+import React, { ReactElement, useState } from 'react';
+import {
+    Divider,
+    Flex,
+    FlexItem,
+    Stack,
+    StackItem,
+    Text,
+    Title,
+    ToggleGroup,
+    ToggleGroupItem,
+} from '@patternfly/react-core';
+
+import { ExternalEntitiesIcon } from '../common/NetworkGraphIcons';
+import ExternalFlowsTable from './ExternalFlowsTable';
+import { CustomEdgeModel, CustomNodeModel } from '../types/topology.type';
+import { getNodeById } from '../utils/networkGraphUtils';
+
+export type ExternalEntitiesView = 'external-ips' | 'workloads-with-external-flows';
+
+export type ExternalEntitiesSideBarProps = {
+    labelledById: string;
+    id: string;
+    nodes: CustomNodeModel[];
+    edges: CustomEdgeModel[];
+    onNodeSelect: (id: string) => void;
+};
+
+function ExternalEntitiesSideBar({
+    labelledById,
+    id,
+    nodes,
+    edges,
+    onNodeSelect,
+}: ExternalEntitiesSideBarProps): ReactElement {
+    const [selectedView, setSelectedView] = useState<ExternalEntitiesView>('external-ips');
+    const entityNode = getNodeById(nodes, id);
+
+    return (
+        <Stack>
+            <StackItem>
+                <Flex direction={{ default: 'row' }} className="pf-v5-u-p-md pf-v5-u-mb-0">
+                    <FlexItem>
+                        <ExternalEntitiesIcon />
+                    </FlexItem>
+                    <FlexItem>
+                        <Title headingLevel="h2" id={labelledById}>
+                            {entityNode?.label}
+                        </Title>
+                        <Text className="pf-v5-u-font-size-sm pf-v5-u-color-200">
+                            Connected Entities Outside Your Cluster
```

Comment:
Here is an example of same text in **Sentence case**:
https://github.com/stackrox/stackrox/blob/master/ui/apps/platform/src/Containers/NetworkGraph/external/ExternalGroupSideBar.tsx#L77

Reply:
Updated to sentence case here and `TopologyComponent.tsx`